templates/fr/complaint.txt

```diff
@@ -0,0 +1,13 @@
+À qui de droit :
+
+Je suis, par les présentes, le dépôt d'une plainte conformément à l'Article 77 RGPD. Sur {request_date}, j'ai envoyé une demande conformément à l'Article {request_article} RGPD pour le contrôleur suivant:
+{request_recipient_address}
+
+{Description des faits par le contrôleur et les étapes que tu avez pris (par exemple, injonctions ou toute autre correspondance, inclure les reçus)}. 
```

Comment:
I also only have very basic knowledge of French but shouldn't this be `tu as` then?

Reply:
I think it should either be `tu as` or `vous avez`.

Reply:
adding the corrected translation